tap_github/streams.py

```diff
@@ -370,6 +372,7 @@ def sync_endpoint(self,
         bookmark_time = singer.utils.strptime_to_utc(min_bookmark_value)
 
         # Build full url
+        self.url = client.domain
```

Comment:
I think the client does not have any attribute name called `domain`

Reply:
Removed unnecessary `domain` attribute from the class.